File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/domain/Orders/Events/OrderPaymentConfirmed.cs
using System;

namespace Registration.Orders
{
    public class OrderPaymentConfirmed : OrderEvent
    {
        public OrderStatus OrderStatus { get; private set; }

        public OrderPaymentConfirmed() { }
        public OrderPaymentConfirmed(Guid conferenceId, OrderStatus orderStatus)
            : base(conferenceId)
        {
            OrderStatus = orderStatus;
        }
    }
}

File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/domain/orders/events/OrderPaymentConfirmed.java
package com.microsoft.conference.registration.domain.orders.events;

import com.microsoft.conference.registration.domain.orders.models.OrderStatus;

public class OrderPaymentConfirmed extends OrderEvent {
    public OrderStatus orderStatus;

    public OrderPaymentConfirmed() {
    }

    public OrderPaymentConfirmed(String conferenceId, OrderStatus orderStatus) {
        super(conferenceId);
        this.orderStatus = orderStatus;
    }
}
